src/Tokens/DatabaseTokenRepository.php

```diff
@@ -85,7 +85,7 @@ protected function getPayload(string $mobile, string $token): array
     public function exists(MustVerifyMobile $user, string $token): bool
     {
         $record = (array) $this->getTable()
-            ->where($user->getMobileField(), $user->getMobileForVerification())
```

Comment:
Please private the `getMobileField()`  [method from this line](https://github.com/mohammad-fouladgar/laravel-mobile-verification/blob/8b9ef1b48717304560b59096d5fb1623a9bc1ac0/src/Concerns/MustVerifyMobile.php#L38) and also remove it from [here](https://github.com/mohammad-fouladgar/laravel-mobile-verification/blob/8b9ef1b48717304560b59096d5fb1623a9bc1ac0/src/Contracts/MustVerifyMobile.php#L42).